libraries/AP_RCProtocol/AP_RCProtocol.h

```diff
@@ -169,7 +169,7 @@ class AP_RCProtocol {
     void add_uart(AP_HAL::UARTDriver* uart);
     bool has_uart() const { return added.uart != nullptr; }
 
-#ifdef IOMCU_FW
+#if AP_RCPROTOCOL_ENABLE_SET_RC_PROTOCOLS
```

Comment:
we could just remove the ifdef completely and let the compiler elide when its not used

Reply:
> we could just remove the ifdef completely and let the compiler elide when its not used

After letting the compiler do the eliding:
```
Board               AP_Periph  blimp  bootloader  copter  heli  iofirmware  plane  rover  sub
Durandal                       *      *           *       *                 *      *      *
HerePro             *                                                                     
Hitec-Airspeed      *                                                                     
KakuteH7-bdshot                *      *           *       *                 *      *      *
MatekF405                      *      *           *       *                 *      *      *
Pixhawk1-1M-bdshot             *                  *       *                 *      *      *
f103-QiotekPeriph   *                                                                     
f303-Universal      *                                                                     
iomcu                                                           *                         
revo-mini                      *      *           *       *                 *      *      *
skyviper-v2450                                    *                                       
```